diff --git a/src/con_duct/__main__.py b/src/con_duct/__main__.py
index 1926a9e..2c23489 100755
--- a/src/con_duct/__main__.py
+++ b/src/con_duct/__main__.py
@@ -463,6 +463,7 @@ class Report:
             output = subprocess.check_output(
                 [
                     "ps",
+                    "-w",
                     "-s",
                     str(self.session_id),
                     "-o",
diff --git a/test/test_e2e.py b/test/test_e2e.py
index d222135..878b0e1 100644
--- a/test/test_e2e.py
+++ b/test/test_e2e.py
@@ -29,7 +29,7 @@ def test_spawn_children(temp_output_dir: str, mode: str, num_children: int) -> N
         pid
         for sample in all_samples
         for pid, proc in sample["processes"].items()
-        if proc["cmd"].startswith("sleep")
+        if "sleep" in proc["cmd"]
     )
     # Add one pid for the hold-the-door process, see spawn_children.sh line 7
     if mode == "setsid":
