diff --git a/src/con_duct/__main__.py b/src/con_duct/__main__.py
index 1926a9e..b1b7506 100755
--- a/src/con_duct/__main__.py
+++ b/src/con_duct/__main__.py
@@ -489,7 +489,12 @@ class Report:
                         ),
                     )
         except subprocess.CalledProcessError as exc:  # when session_id has no processes
-            lgr.debug("Error collecting sample: %s", str(exc))
+            lgr.warning("Error collecting sample: %s", str(exc))
+            lgr.warning("cmd:", exc.cmd)
+            lgr.warning("returncode:", exc.returncode)
+            traceback.print_exc(file=sys.stderr)
             return None

         sample.averages = Averages.from_sample(sample)
